config/clients/js/template/configuration.mustache

```diff
@@ -63,10 +62,10 @@ export class Configuration {
    * @type {string}
    * @memberof Configuration
    */
-  private static sdkVersion = "{{packageVersion}}";
```

Comment:
Were the changes to hard-coded values intentional here?

Reply:
Good catch! fixed